maintainers/maintainer-list.nix

```diff
@@ -6428,6 +6428,12 @@
     githubId = 1973389;
     name = "Reuben D'Netto";
   };
+  reardencode = {
+    email = "freedom@reardencode.com";
+    github = "reardencode";
+    githubId = 730881;
+    name = "Brandon Black";
+  };
```

Comment:
Please make this into a separate commit:

````
maintainers: add <name>
```

Reply:
done